db/migrate/20180724144925_remove_unique_index_on_priority.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUniqueIndexOnPriority < ActiveRecord::Migration[5.1]
+  def up
+    remove_index :discovery_rules, :priority
+  end
+
+  def down
+    add_index :discovery_rules, :priority, unique: true
+  end
+end
```

Comment:
I am pretty sure Rails devs will deprecate this soon, there is some new syntax I think...

Reply:
What I mean, use Rails `change` method instead.

Reply:
Going back in time a bit, on IRC you said "one question related to the same, so if i use the `change` method for migration, then while rollback it wont use the `unique: true`". What exactly was the problem? This should work:

    add_index :discovery_rules, column: :priority, unique: true

Reply:
@lzap i guess I had misunderstood the change method, only if there is specific `query` that needs to done we should probably go for the up and down methods, else change should do the work. thanks!

Reply:
Why this is removing priority index? Don't we use it during provisioning when performing auto provisioning?